ops/travis/staging/deploy

```diff
@@ -2,3 +2,5 @@
 # these variable must be available securely from travis
 echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 make travis-push-staging-images
+
+# TODO: PC->SAN: why not moving `travis-push-staging-images` makefile command directly here?
```

Comment:
good point. at the moment I preferred keeping all these things centralized in the makefile.